lib/chef-acceptance/application.rb

```diff
@@ -16,12 +16,14 @@ class Application
     WORKER_POOL_SIZE = 3
 
     attr_reader :force_destroy
+    attr_reader :timeout
```

Comment:
Setting the timeout here is fine, but I don't think you need to update all methods to support passing in `timeout.`  Just use it or worse case, `run_suite` and `run_command` need an options hash.  I don't think that's necessary though.